src/components/core/version.js

```diff
@@ -108,6 +108,11 @@ const Warning = ({ selectedVersion, versionColor, versions }) => (
           API. For the rest, refer to v3.1.
         </p>
       )}
+      {selectedVersion === versions.LEGACY_VERSION_1 && (
```

Comment:
See below, you could use something similar with `includes`

Reply:
Discussed to do it differently